OTVision/pre_annotation.py

```diff
@@ -124,12 +124,14 @@ def main(
     img_type: str = "png",
 ):
     log.info("Starting")
-    if Path(file).is_file():
-        _pre_annotate(file, model_weights, chunk_size, filter_classes, img_type)
-    elif Path(file).is_dir():
-        zip_files = get_files(file, "zip")
-        for file in progressbar.progressbar(zip_files):
-            _pre_annotate(file, model_weights, chunk_size, filter_classes, img_type)
+    f = Path(file)  # convert to Path object if file is of type str
```

Comment:
Should we enforce Path?

Reply:
If so, we dont need conversion to Path. Then we also would not need "f" and keep working with "file", which is a more understandable variable name

Reply:
Correct! Changed with the latest commit~